package.json

```diff
@@ -38,5 +41,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "lint-staged": {
+    "*.{js, jsx}": [
+      "node_modules/.bin/eslint --max-warnings=0",
+      "prettier --write",
+      "git add"
+    ]
   }
```

Comment:
```
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "linters": {
      "*.js": [
        "prettier --write",
        "eslint --fix",
        "git add"
      ],
      "*.{css,scss,less,md,json}": [
        "prettier --write",
        "git add"
      ]
    },
    "ignore": [
      "node_modules",
      "dist",
      "package-lock.json"
    ]
  },
```

Reply:
Add this in package.json file, this will run the husky hooks (I mean, prettier and eslint) before you commit your changes

Reply:
Bug fixed use hint https://github.com/aws/aws-sdk-js-v3/commit/a628140e62af0e3f145061714a1935aaacde800d